browser/extensions/BUILD.gn

```diff
@@ -36,6 +36,8 @@ source_set("extensions") {
     "brave_theme_event_router.h",
     "brave_tor_client_updater.cc",
     "brave_tor_client_updater.h",
+    "brave_wallet_navigation_throttle.cc",
```

Comment:
shouldn't this be in `brave_wallet_enabled`?

Reply:
fixed.